testing/core/var_trans_test.py

```diff
@@ -38,3 +38,10 @@ def test_softplus_numerical(self, x, rtol, atol):
         npt.assert_allclose(mf_pos.asnumpy(), np_pos, rtol=rtol, atol=atol)
         npt.assert_allclose(mf_inv.asnumpy(), np_inv, rtol=rtol, atol=atol)
         npt.assert_allclose(mf_inv.asnumpy(), x.asnumpy(), rtol=rtol, atol=atol)
+
+    def test_logistic(self):
```

Comment:
Can you add some parameterizations to this like in the test_softplus_numerical above? Fine to not compare against numpy, just to test it against a few values.